org-code-javabuilder/lib/src/main/java/org/code/javabuilder/ProjectFileLoader.java

```diff
@@ -0,0 +1,66 @@
+package org.code.javabuilder;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.concurrent.Callable;
+
+/**
+ * Fetches an individual file from a student's project. This should be used with the
+ * {dashboardHost}/v3/files/{channelId}/{fileName} API. It has no permissions to use AWS S3. This
+ * should be used asynchronously so multiple files can be retrieved at once.
```

Comment:
Maybe clarify this last sentence to say that this class is intended to be called by an executor?  (The code here doesn't really have a requirement to be called asynchronously.)

Reply:
✔️